WORKSPACE

```diff
@@ -39,9 +39,9 @@ new_local_repository(
 # 2. Update .bazelrc and .bazelversion files.
 #
 # envoy commit date: 01/03/2020
```

Comment:
nit: maybe remove this as we're likely not going to keep it up to date?

Reply:
done